pom.xml

```diff
@@ -141,8 +141,8 @@
     <lombok.version>1.16.22</lombok.version>
     <lz4.version>1.3.0</lz4.version>
     <mockito.version>2.13.0</mockito.version>
-    <netty.version>4.1.22.Final</netty.version>
-    <netty-boringssl.version>2.0.7.Final</netty-boringssl.version>
+    <netty.version>4.1.28.Final</netty.version>
```

Comment:
We should log netty upgrade in a different commit please, so it will be easier to notice that important change.

Reply:
I upgraded netty because older version produced empty stack trace for some leaks. 
I'll revert the version upgrade in this change.